src/functionalTest/java/uk/gov/hmcts/reform/professionalapi/FindUsersByOrganisationTest.java

```diff
@@ -1,283 +1,298 @@
-            assertThat(user.get("lastName")).isNotNull();
-            assertThat(user.get("email")).isNotNull();
-            if (!expectedStatus.equals("any")) {
-                assertThat(user.get("idamStatus").equals(expectedStatus));
-            }
-            if (user.get("idamStatus").equals(IdamStatus.ACTIVE.toString()) && rolesReturned) {
-                assertThat(user.get("roles")).isNotNull();
-            } else {
-                assertThat(user.get("roles")).isNull();
-            }
-        });
-    }
-}
+package uk.gov.hmcts.reform.professionalapi;
+
+import static org.apache.commons.lang.RandomStringUtils.randomAlphabetic;
+import static org.assertj.core.api.Assertions.assertThat;
+import static uk.gov.hmcts.reform.professionalapi.controller.request.NewUserCreationRequest.aNewUserCreationRequest;
+import static uk.gov.hmcts.reform.professionalapi.helper.OrganisationFixtures.createJurisdictions;
+
+import io.restassured.specification.RequestSpecification;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import lombok.extern.slf4j.Slf4j;
+import net.serenitybdd.junit.spring.integration.SpringIntegrationSerenityRunner;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.http.HttpStatus;
+import org.springframework.test.context.ActiveProfiles;
+import uk.gov.hmcts.reform.professionalapi.controller.constants.IdamStatus;
+import uk.gov.hmcts.reform.professionalapi.controller.request.NewUserCreationRequest;
+
+
+@RunWith(SpringIntegrationSerenityRunner.class)
+@ActiveProfiles("functional")
+@Slf4j
+public class FindUsersByOrganisationTest extends AuthorizationFunctionalTest {
+
+    RequestSpecification bearerTokenForExternalRole;
+    RequestSpecification bearerTokenForNonPuiUserManager;
+
+
+    public RequestSpecification generateBearerTokenForExternalRole(String role) {
+        Map<String, Object> response = professionalApiClient.createOrganisation();
+        String orgIdentifierResponse = (String) response.get("organisationIdentifier");
+        professionalApiClient.updateOrganisation(orgIdentifierResponse, hmctsAdmin);
+
+        List<String> userRoles = new ArrayList<>();
+        userRoles.add(role);
+        String userEmail = randomAlphabetic(5).toLowerCase() + "@hotmail.com";
+        String lastName = "someLastName";
+        String firstName = "someName";
+
+        bearerTokenForExternalRole = professionalApiClient.getMultipleAuthHeadersExternal(role, firstName, lastName, userEmail);
+
+        NewUserCreationRequest userCreationRequest = aNewUserCreationRequest()
+                .firstName(firstName)
+                .lastName(lastName)
+                .email(userEmail)
+                .roles(userRoles)
+                .jurisdictions(createJurisdictions())
+                .build();
+        professionalApiClient.addNewUserToAnOrganisation(orgIdentifierResponse, hmctsAdmin, userCreationRequest, HttpStatus.CREATED);
```

Comment:
same here also

Reply:
Please can you apply hidden whitespaces setting and review. I have tried by reverting the change but no effect. I will take care this next time. Thanks.